doctr/datasets/loaders/detection/core.py

```diff
@@ -54,6 +54,8 @@ def __init__(
         labels_path: str,
         batch_size: int = 1,
         shuffle: bool = True,
+        std_rgb: Tuple[float, float, float] = (0.264, 0.274, 0.287),
+        std_mean: Tuple[float, float, float] = (0.798, 0.785, 0.772),
```

Comment:
Typo here 
Let's name them mean & std adn specify in the docstring that they are in RGB order